src/wrapper/signature.rs

```diff
@@ -79,15 +78,19 @@ pub struct TypeSignature {
 
 impl TypeSignature {
     /// Parse a signature string into a TypeSignature enum.
+    #[allow(clippy::should_implement_trait)]
+    /// Clippy suggests implementing `FromStr` or renaming it which is not possible in our case.
```

Comment:
Here as well, this is visible in docs (it shouldn't) and is placed after line it describes, not before, which is weird.